### PR TITLE
v1.2.2

### PR DESCRIPTION
**Pull request by jjr840430**

### CODE REVIEW

**Comment by ronaldtse** [comment]
@jjr840430 , instead of changing command syntax (`reload`) into argument options (`--reload`), can you allow libnereon to have an additional syntax that allows a "command", like:

* `[exec] [command] [arg-option, ...]`
* `[exec] [command] [subcommand] [arg-option, ...]`
?

**Comment by jjr840430** [comment]
@ronaldtse `rvc` may use options without prefix.

File: tests/rvd_tests.c [hunk 1]
```diff
@@ -421,7 +421,7 @@ permission_check_ovpn_bin(void)
 	pid_t rvd_pid;
 	int exit_code;
 
-	char *args[] = {RVC_BIN_PATH, "test", "connect", "--json", NULL};
+	char *args[] = {RVC_BIN_PATH, "conect", "test", "json", NULL};
```
Comment (ronaldtse) [inline tests/rvd_tests.c]: `connect` not `conect`?

Comment (jjr840430) [reply tests/rvd_tests.c]: @ronaldtse fixed.

**Pull request merged**